### PR TITLE
Add support for mouse wheel zoom

### LINKED ISSUE #31: Increase font size in mini-mode
Currently the mini UI is hard to read on 4k displays. Instead of futzing around with trying to find a font size that works, or doing different scaling based on screen resolution (is that even possible?), mouse wheel zoom would solve the issue and give users control over how big they want stuff.

### PR DESCRIPTION
**Pull request by neilenns**
Fixes #31 

Adds an event handler for the `zoom-changed` event and uses it to adjust the zoom level for the window contents, limited to between 100% and 300%.

This applies to the entire window contents, so it does zoom the non-mini UI as well. I'm not aware of a way to only zoom if the mini content is what's shown.

### CODE REVIEW

File: src/app/components/mini.tsx [hunk 1]
```diff
@@ -19,7 +19,9 @@ const Mini: React.FC = () => {
                 </span>
                 :{" "}
                 <span
-                  style={{ color: radio.currentlyRx ? "green" : "inherit" }}
+                  style={{
+                    color: radio.currentlyRx ? "lightgreen" : "inherit",
```
Comment (neilenns) [inline src/app/components/mini.tsx:23]: Unrelated change 😬 I found "green" was too dark against the dark background. Let me know if you'd like this split out to a separate PR instead.

Comment (pierr3) [reply src/app/components/mini.tsx:23]: Approved

**Comment by pierr3** [comment]
I'll admit, I am not a huge fan of this. Zooming in to the UI of a desktop application is clunky. Users will not that they accidentally zoomed in or a lot of cases, and will open issues "why is my UI zoomed in". I'd rather we properly support high DPI.

**Comment by neilenns** [comment]
> I'll admit, I am not a huge fan of this. Zooming in to the UI of a desktop application is clunky. Users will not that they accidentally zoomed in or a lot of cases, and will open issues "why is my UI zoomed in". I'd rather we properly support high DPI.

Fair. Maybe just a setting for increased font size? Or maybe just make the font size bigger in mini mode? The main mode is fine honestly.

**Comment by neilenns** [comment]
Will open a different PR with just an increased font size per discussion.
